conda/conda_build_config_attempt.yaml

```diff
@@ -0,0 +1,3 @@
+qt:
+    - '<5'
+    - '>4'
```

Comment:
Add a new line at the end of the file.

Reply:
this was an attempt to create variant. Doesnt look mature enough or I didnt understood well how it works.